docs/DeveloperGuide.md

```diff
@@ -476,10 +476,9 @@ testers are expected to do more *exploratory* testing.
 
 1. Initial launch
 
-   1. Download the jar file and copy into an empty folder
+   1. Download the jar file and copy into an empty folder.
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample friends. The window size may not be 
-      optimum.
+   1. Double-click the jar file Expected: gitGud GUI launches.
 
 1. Saving window preferences
 
```

Comment:
Can you help remove `the resize window to an optimum size`. We can't resize it anymore.